bigtable/tests/system.py

```diff
@@ -197,6 +197,7 @@ def test_create_instance(self):
         self.assertEqual(instance, instance_alt)
         self.assertEqual(instance.display_name, instance_alt.display_name)
         self.assertEqual(instance.type_, instance_alt.type_)
+        self.assertIsInstance(instance_alt.labels, dict)
         self.assertEqual(instance.labels, instance_alt.labels)
```

Comment:
It would be better to assert equality with `LABELS` (we want to know that they were assigned correctly).  That would cover the `isinstance` bit, too.

Reply:
The test against an explicit dict literal accomplishes *both* goals:  the dict values are correct, *and* `instance.lables` returns a dict, not a proto message.

Reply:
Thank you for the advice. It is done